Assignment2/Assignment2_QNo3/SolAssignmentNo2_QNo3.rb

```diff
@@ -0,0 +1,28 @@
+# => 3 Write an example of how you can use super method with and without arguments.
+# => Super Parent Class
+class SuperParentSkeleton
+  def initialize
+    puts "Hii You chose option : Super without Arguments in Parent Class"
+  end
+  def super_argument(num)
+    puts "hii You chose option : Super with Arguments in Parent Class"
+  end
+end
+
+# => Super child class
+class SuperChildSkeleton < SuperParentSkeleton
+  def initialize
+    # super without arguments
+    super
+    puts "Hii You chose option : Super without Arguments in Child Class"
+  end
```

Comment:
@HrisikeshSinghaChowdhury Write an example where both parent and child method is having same argument list (instead of no argument) and then you call super method inside child method.

Reply:
here i have used both methods with same name in both parent and child class
1) initialize(without argument)
2) super_argument(with argument)
Note : According to my understanding